opentracing-tracerresolver-itest/src/test/java/io/opentracing/contrib/tracerresolver/Mocks.java

```diff
@@ -75,4 +78,33 @@ public Tracer convert(Tracer existingTracer) {
         }
     }
 
+    /**
```

Comment:
Although MockTracer isn't actually used, not sure I see the benefit of having your own local impl vs just using the mock tracer as a test dependency?

Reply:
Great point. I was rather unhappy about this, but the integration test is for when there is no `GlobalTracer` on the classpath.
Since v0.31.0, the `MockTracer` has a dependency on `opentracing-util`, hence the `GlobalTracer` will be on the classpath.
I tried using the `MockTracer` with excluded dependency on _util_, but that failed due to missing `ScopeManager` implementation.

Since we don't actually _use_ the tracers, but merely verify their registration, a basic unimplemented interface was the least complex solution I could find.

Reply:
Ok understand.